### DISCUSSION TITLE
Ошибка просмотра таблицы после изменения имени поля, по которому производится сортировка

### DISCUSSION THREAD

**Discussion by nshy** [totumonline/totum-ru-issues-and-discussions#569] [ОШБК!] [answered]
Собственно после изменения поля видно только PHP ошибку. Саму таблицу не видно.

Вылечил изменив поле сортировки для таблицы через системные таблицы.

`totum-mit 4.11.57.0`.

**Comment by ttm-alexeypolunin** [comment, marked as answer]
Да. Это известная ошибка. Стоит в списке на обработку. Спс
